applets/cosmic-app-list/Cargo.toml

```diff
@@ -5,33 +5,29 @@ edition = "2021"
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 [dependencies]
-cosmic-panel-config = {git = "https://github.com/pop-os/cosmic-panel", features = ["gtk4"] }
-cosmic-protocols = { git = "https://github.com/pop-os/cosmic-protocols", default-features = false, features = ["client"] }
-cascade = "1.0.0"
-gtk4 = { git = "https://github.com/gtk-rs/gtk4-rs", features = ["v4_4"] }
-adw = { git = "https://gitlab.gnome.org/World/Rust/libadwaita-rs", package = "libadwaita"}
-libcosmic = { git = "https://github.com/pop-os/libcosmic", default-features = false }
-gio = { git = "https://github.com/gtk-rs/gtk-rs-core" }
-relm4-macros = { git = "https://github.com/Relm4/Relm4.git", branch = "next" }
-serde_json = "1.0.78"
-futures = "0.3.19"
-futures-util = "0.3.19"
-once_cell = "1.9.0"
-xdg = "2.4.0"
-gsk4 = { git = "https://github.com/gtk-rs/gtk4-rs" }
+cctk = {path = "../../../cosmic-protocols/client-toolkit", package = "cosmic-client-toolkit"}
+# cosmic-protocols = { git = "https://github.com/pop-os/cosmic-protocols", default-features = false, features = ["client"] }
+cosmic-protocols = { path = "../../../cosmic-protocols", default-features = false, features = ["client"] }
```

Comment:
Currently this is depending on cosmic-protocols existing in the parent directory.